content/source/docs/extend/best-practices/sensitive-state.html.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: "extend"
+page_title: "Extending Terraform: Sensitive State Best Practices"
+sidebar_current: "docs-extend-best-practices-sensitive-state"
+description: |-
+  Recommendations for handling sensitive information in state.
+---
+
+# Handling Sensitive Values in State
+
+Many organizations use Terraform to manage their entire infrastructure, and
+it's inevitable that [sensitive information will find its way into
+Terraform](/docs/state/sensitive-data.html) in these circumstances. There are a
+couple of recommended approaches for managing sensitive state in Terraform.
+
+## Using the `Sensitive` Flag
+
+When working with a field that is likely to contain sensitive information, it
+is best to set the
+[`Sensitive`](https://godoc.org/github.com/hashicorp/terraform-plugin-sdk/helper/schema#Schema.Sensitive)
+property on its schema to `true`. This will prevent the field's values from
+showing up in CLI output and in Terraform Cloud. It will not encrypt or obscure
+the value in the state, however.
+
+## Storing Only a Hash
+
+Providers can also protect sensitive information by simply not storing it.
```

Comment:
Technically we are storing a hash of it and hash could be considered as sensitive, especially if it's something like MD5 or SHA1.